packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -21,12 +22,19 @@ export class CheckboxBase extends React.Component<ICheckboxProps, ICheckboxState
   private _classNames: { [key in keyof ICheckboxStyles]: string };
 
   public static getDerivedStateFromProps(props: ICheckboxProps, state: ICheckboxState): ICheckboxState {
+    if (!props.defaultIndeterminate && state.isIndeterminate) {
+      return {
+        ...state,
```

Comment:
>...state, [](start = 8, length = 9)

I actually don't know for sure but is this spread really necessary? I do recognize that you probably followed the existing pattern in lines 32-36 though..

Reply:
@KevinTCoughlin  looked into this and validated it isn't necessary

https://github.com/facebook/react/blob/d9b4c55d53c67045c38b12f5aad57a163e7f63a2/packages/react-reconciler/src/ReactFiberClassComponent.js#L163

Reply:
True, I was following suit with the existing pattern but after removing it and checking it now it works so i'll drop both in both scenarios.